server/test/controllers/applicant/UpsellControllerTest.java

```diff
@@ -44,6 +45,7 @@ public void considerRegister_redirectsToUpsellViewForDefaultProgramType() {
     ApplicantModel applicant = createApplicantWithMockedProfile();
     ApplicationModel application =
         resourceCreator.insertActiveApplication(applicant, programDefinition.toProgram());
+    application.setSubmitTimeForTest(Instant.ofEpochSecond(0));
```

Comment:
**suggestion:** We could consider pulling this out to a variable, like is done [here](https://github.com/civiform/civiform/blob/2c03055565eaa09ce521d143dbba3cecc348e292/server/test/services/export/AbstractExporterTest.java#L57C3-L57C91)